scripts/lint

```diff
@@ -2,7 +2,6 @@
 
 cd "$(dirname "$0")/.."
 
-python3 -m black --check hass_nabucasa tests setup.py
-python3 -m flake8 hass_nabucasa setup.py tests
-python3 -m pylint --rcfile pylintrc hass_nabucasa
+python3 -m black --check --target-version py38 hass_nabucasa tests
```

Comment:
3.8 is old and not really in use, but that is whats defined in the vscode definitions, I will do a PR to bump that later